libs/libupm/Makefile

```diff
@@ -51,13 +51,14 @@ UPM_MODULES:= \
 	veml6070 water waterlevel wfs wheelencoder wt5001 xbee yg1006 zfm20 \
 	vcap t3311 hwxpxx h803x ozw curieimu
 # (require libbacnet) tb7300 t8100 e50hx bacnetmstp
+# (require libtinyb) 2jciebu01_ble 2jciebu01_usb
 
 define Package/libupm/Default
   SECTION:=libs
   CATEGORY:=Libraries
   SUBMENU:=IoT
   URL:=https://github.com/intel-iot-devkit/upm
-  DEPENDS:=+libmraa +librt
+  DEPENDS:=@!arc @!armeb @!powerpc +libmraa +librt
```

Comment:
Those dependencies are implicit because of libmraa.

Reply:
fix